### ISSUE TITLE
Search for author/owner only

### ISSUE THREAD

**Issue by keunes** [Podcastindex-org/docs-api#50]
I was just thinking of a practical use-case for [AntennaPod](https://github.com/AntennaPod/AntennaPod): it would be great if in the 'add podcast' screen, one could tap on a podcast, preview an episode, get hooked, subscribe and then tap on the Author to discover more podcasts from the same publisher (e.g. public broadcaster channel, big news outlet).

Of course AntennaPod could just take the author name (that's displayed in the app) and use it for general search. However, currently searching for `Radio France` will also return podcasts from the BBC (on the Tour de France) and others.

Being able to check only the `author`+`ownerName` fields would be helpful.

(Sidenote: thanks @stevencrader for making the Postman Collection and Environment available! Not a developer at all and Postman is a great way to start experimenting with the API!)

**Comment by daveajones** [comment]
That would be challenging atm, but I can create it with some adjustment to the way the search indexer currently works.  I'll put on the to-do list.

**Comment by keunes** [comment]
Makes me think: would/should it be a separate endpoint? Just like there is `byperson`.

At the same time, if in the client I'd want to do an advanced search (e.g. `person=Adam Curry&category=technology`, or `language=nl&category=news`), it'd all have to be in a single end-point, no?

**Comment by daveajones** [comment]
Yeah, i'd like to keep it all under normal search with just some more params like that.

**Comment by keunes** [comment]
Related: #80

**Comment by daveajones** [comment]
@keunes Can you check the search/byperson endpoint and see if it now would meet your criteria for this?  I've changed it significantly to do better matching.  The results should have all of the feed metadata.  Let me know.
